torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -166,6 +166,20 @@ RegisterOperators reg({
             return 0;
           };
         }),
+    Operator(
+        prim::NumToBool,
+        [](Node* node) -> Operation {
+          return [](Stack& stack) {
+            at::Scalar s;
+            pop(stack, s);
+            if (s.isIntegral()) {
+              push(stack, s.toLong() != 0);
+            } else {
+              push(stack, s.toDouble() != 0);
```

Comment:
Will this comparison actually do what you want? https://randomascii.wordpress.com/2012/02/25/comparing-floating-point-numbers-2012-edition/

Reply:
Python doesn't do any rounding it looks like so this should be fine.

```
>>> bool(.3 - .2 - .1)
True
```

Reply:
lol k